Solutions/Menes.Abstractions/Menes/OpenApiPathTemplate.cs

```diff
@@ -18,11 +18,13 @@ public class OpenApiPathTemplate
         /// </summary>
         /// <param name="uriTemplate">The uri template.</param>
         /// <param name="item">The path item.</param>
-        public OpenApiPathTemplate(string uriTemplate, OpenApiPathItem item)
+        /// <param name="document">The Open API document in which this is a path template.</param>
+        public OpenApiPathTemplate(string uriTemplate, OpenApiPathItem item, OpenApiDocument? document)
         {
             this.UriTemplate = new UriTemplate(uriTemplate);
             this.PathItem = item;
-            this.Match = new Regex(UriTemplate.CreateMatchingRegex(uriTemplate), RegexOptions.Compiled);
+            this.Match = new Regex(UriTemplate.CreateMatchingRegex2(uriTemplate), RegexOptions.Compiled);
```

Comment:
...so you've found the need to change to the `2` form so you _must_ have spotted a substantial difference! But what?

Reply:
"2"